src/test/java/com/google/devtools/build/lib/bazel/repository/PatchUtilTest.java

```diff
@@ -378,7 +408,7 @@ public void testMissingBothOldAndNewFile() throws IOException {
     Path patchFile =
         scratch.file(
             "/root/patchfile",
-            "diff --git a/foo.cc b/foo.cc",
+            "diff --git a/ b/",
```

Comment:
How is this change related?

Reply:
So this test case is for testing an error message is thrown when no input and output file name is specified. Before, we only parse file names from `---` and `+++` lines. But now we also parses file names from `diff --git` line, therefore we should also remove them here to keep the test passing.